### ISSUE TITLE
Gallery - Console Error: Failed to load resource: net: : ERR_BLOCKED_BY_CLIENT

### ISSUE THREAD

**Issue by romatitan** [titan-cebu/BNT-Legal#74]
![image](https://cloud.githubusercontent.com/assets/12888048/9515680/58874e14-4cd5-11e5-905b-89e5555db76e.png)

**Comment by christiancebudigital** [comment]
Please clear Cache..  it is working well on our end..

**Comment by romatitan** [comment]
I've cleared the cache and check again the console.

Chrome - console error: Falied to load resource: net::ERR_BLOCKED_BY_CLIENT
The error still occurs at my end, but I've tried to check with other PC and chrome-browser and there's no error occurs.
Result: 1:3
For chrome browser its won't fix.

But still a console error occurs in IE browsers.
IE - console error: SCRIPTS: Access is denied.
Result: 3:3
![image](https://cloud.githubusercontent.com/assets/12888048/9538579/9485921a-4d78-11e5-97ba-15ef7f84076c.png)
